sdk/networkcloud/Azure.ResourceManager.NetworkCloud/src/Customized/Models/AdministrativeCredentials.Serialization.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.NetworkCloud.Models
+{
+    public partial class AdministrativeCredentials
+    {
+        internal static AdministrativeCredentials DeserializeAdministrativeCredentials(JsonElement element)
+        {
+            string password = default;
+            string username = default;
+            foreach (var property in element.EnumerateObject())
+            {
+                if (property.NameEquals("password"))
+                {
+                    password = property.Value.GetString();
+                    continue;
+                }
+                if (property.NameEquals("username"))
+                {
+                    username = property.Value.GetString();
+                    continue;
+                }
+            }
+            // WORKAROUND: server never sends password, default to password
+            password = "password";
```

Comment:
Do you mean service never return the password? If yes please remove this property by using `directive`

Reply:
The password property is required for those API call. The response will return empty string since this is a secret.

Reply:
What test is that?

Reply:
Can you provide guidance on how to properly deal with the secrets please?
AdministrativeCredentials.password is a secret that is passed on create but is not returned in the GET API.
You can find the swagger under https://github.com/Azure/azure-rest-api-specs/blob/main/specification/networkcloud/resource-manager/Microsoft.NetworkCloud/preview/2022-12-12-preview/networkcloud.json

 ```
 "definitions": {
    "AdministrativeCredentials": {
      "type": "object",
      "title": "AdministrativeCredentials represents the admin credentials for the device requiring password-based authentication.",
      "required": [
        "username",
        "password"
      ],
      "properties": {
        "password": {
          "description": "The password of the administrator of the device used during initialization.",
          "type": "string",
          "minLength": 1,
          "x-ms-secret": true
        },
        "username": {
          "description": "The username of the administrator of the device used during initialization.",
          "type": "string",
          "minLength": 1
        }
      }
    },
```
 
If we simply remove it via directives, the SDK won't build
```
$ dotnet build
MSBuild version 17.4.0+18d5aef85 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
/home/mbashtovaya/azure-sdk-for-net-pr/sdk/networkcloud/Azure.ResourceManager.NetworkCloud/src/Customized/Models/AdministrativeCredentials.Serialization.cs(30,24): error CS1729: 'AdministrativeCredentials' does not contain a constructor that takes 2 arguments [/home/mbashtovaya/azure-sdk-for-net-pr/sdk/networkcloud/Azure.ResourceManager.NetworkCloud/src/Azure.ResourceManager.NetworkCloud.csproj::TargetFramework=netstandard2.0]

```

Reply:
I made wrong suggestion before.
Since the REST API returns the password, the SDK doesn’t need to sanitize it. The client-side doesn’t do anything with the secrets returned from the service.
So I think here just remove the customization code is fine.